src/Kunstmaan/AdminListBundle/Service/ExportService.php

```diff
@@ -78,7 +78,7 @@ protected function streamOutput(ExportableInterface $adminList, $format)
 
             $iterator = $adminList->getIterator();
             foreach ($iterator as $item) {
-                if (\array_key_exists(0, $item)) {
+                if (\is_array($item) && \array_key_exists(0, $item)) {
```

Comment:
See also https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/3421, so I think it can be simplified to `$itemObject = $item;`